L/liblinear/build_tarballs.jl

```diff
@@ -17,7 +17,7 @@ cd $WORKSPACE/srcdir
 for f in ${WORKSPACE}/srcdir/patches/*.patch; do
     atomic_patch -p1 ${f}
 done
-cd liblinear-230/
+cd liblinear/
 mkdir -p ${prefix}/bin
 mkdir -p ${prefix}/lib
 if [[ "${target}" == *-freebsd* ]] || [[ "${target}" == *-apple-* ]]; then
```

Comment:
Uhm, I don't remember why were using gcc, it'd be good to see if we can avoid that

Reply:
should I try just deleting this `if` block?

Reply:
trying that in https://github.com/JuliaPackaging/Yggdrasil/pull/7698/commits/724addf649f26a9cea343551d1e0b83c0b17e0b8